index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    
+    <header class="toolbar">
+     
+       <nav class="logo"><a  href="#"><img src="Images\Logo Placeholder.png" alt="logo" /></a></nav> 
+      <img class="icons" src="Images\Enabled.png" alt="threeline" />
+  </header>
+
+  <section class="sec">
+    <div class="headline">
+    
+        <h1 class="text1">I'm Burak Glad to see you</h1>
+        <p class="text2">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+      
+        <h2><p class="let">LET'S CONNECT</p></h2>
```

Comment:
- [ ] There's no need to wrap a `<p>` tag inside an `<hx>` one. Basically, you're telling the browser that inside a "title" there's a "paragraph" which makes no sense! Please consider leaving it just as a title which is the correct representation of this one. 🤓 🛠️ 

```suggestion
        <h2 class="let">LET'S CONNECT</h2>
```